scss/bootstrap/main.scss

```diff
@@ -67,6 +67,22 @@ tags-input {
     border-color: #843534;
     @include box-shadow(inset 0 1px 1px rgba(0, 0, 0, .075), 0 0 6px #ce8483)
   }
+  .disabled {
```

Comment:
I think it would be better to target the `tag-input` element directly, like this:

```css
tags-input[disabled] {
...
}
```

That way you wouldn't need to set any class to any element.